src/sentry/web/frontend/auth_login.py

```diff
@@ -106,6 +107,11 @@ def get_next_uri(self, request: Request):
         next_uri_fallback = None
         if request.session.get("_next") is not None:
             next_uri_fallback = request.session.pop("_next")
+
+        # If the active org has an ongoing onboarding session, redirect to onboarding after login
+        organization = self.get_active_organization(request)
+        if organization:
+            next_uri_fallback = get_client_state_redirect_uri(organization.slug, next_uri_fallback)
```

Comment:
@Neo-Zhixing can you add unit tests for the redirects?

Reply:
I've added an acceptance test for homepage redirects. I guess I can add unit tests too

Reply:
@Neo-Zhixing oh yea, if you are testing it in acceptance tests that's fine